test/e2e/tests/request-queuing/sendTx-switchChain-sendTx.spec.js

```diff
@@ -0,0 +1,113 @@
+const FixtureBuilder = require('../../fixture-builder');
+const {
+  withFixtures,
+  openDapp,
+  unlockWallet,
+  DAPP_URL,
+  regularDelayMs,
+  WINDOW_TITLES,
+  switchToNotificationWindow,
+  defaultGanacheOptions,
+} = require('../../helpers');
+
+describe('Request Queuing Send Tx -> SwitchChain -> SendTx', function () {
+  it('should not be able to navigate batch send txs with a switch chain in the middle', async function () {
```

Comment:
I think we should set this as failing (expected) `test.failing` (https://jestjs.io/docs/api#testfailingname-fn-timeout) and merge the tests that are passing and come back when we have this fixed